hypothesis/track_hypo.py

```diff
@@ -0,0 +1,287 @@
+import numpy as np
+import math
+from sparse import sparse
+from numba import jit, int32, float32
+
+def PowerAxis(minval, maxval, n_bins, power):
+    l = np.linspace(np.power(minval, 1./power), np.power(maxval, 1./power), n_bins+1)
+    bin_edges = np.power(l, power)
+    return bin_edges
+
+@jit((float32, float32, float32, float32, float32, float32, float32, float32, float32), nopython=True, nogil=True, fastmath=True)
+def numba_bin_indices(t, x, y, z, radius, t_scaling_factor, r_scaling_factor, theta_scaling_factor, phi_scaling_factor):
```

Comment:
add some minimal documentation to each function and module describing what they do (but no need to define every single parameter)